tox.ini

```diff
@@ -1,10 +1,9 @@
 [tox]
 envlist =
-    py27,py35,py36,py37,py38,pypy,pypy3,
-    py27-pyramid{13,14,15,16,17,18,19,110},
-    py37-pyramid{13,14,15,16,17,18,110},
-    docs,
-    {py2,py3}-cover,coverage
+    py36,py37,py38,py39,pypy3,
+    py38-pyramid{13,14,15,16,17,18,110,20},
+    py38-cover,coverage,
```

Comment:
```suggestion
    py39-cover,coverage,
```

Reply:
We previously ran coverage on Python 2 (`py2-cover`), then Python 3 (`py3-cover`), then combined them in a third tox environment (`coverage`). This should be simplified to run coverage once on a Python 3 environment and report on it, instead of two environments.